rockcraft/services/package.py

```diff
@@ -139,17 +136,7 @@ def _pack(
 
     if project.entrypoint_command:
         emit.progress("Setting OCI entrypoint")
-
-        # Check for additional args
-        regexp = re.match(r"^(.*?)\[\s*(.*?)\s*\]\s*$", project.entrypoint_command)
```

Comment:
this regex check is not present in the new `parse_command()` implementation, is it not necessary?